demos/segmentation_demo/README.md

```diff
@@ -1,12 +1,12 @@
-# Image Segmentation C++ Demo
+# Image Segmentation (asynchronous) C++ Demo
```

Comment:
```suggestion
# Image Segmentation C++ Demo
```
let's keep words that demo can run infer requests asynchronously below, in detailed description. The fact that it is segmentation demo is more general. And sync/async mode chosen by number of infer requests (well nireq=1 is only emulate sync mode but with good approximation)

Reply:
Done